aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -824,15 +832,19 @@ Tensor nuclear_norm(const Tensor& self, IntArrayRef dim, bool keepdim) {
 
 Tensor& nuclear_norm_out(Tensor& result, const Tensor& self, IntArrayRef dim, bool keepdim) {
   TORCH_CHECK(dim.size() == 2, "nuclear norm requires a 'dim' argument of size 2");
+  auto dim_ = dim.vec();
+  maybe_wrap_dims(dim_, self.dim());
 
-  auto permutation = create_dim_backshift_permutation(dim[0], dim[1], self.dim());
+  auto permutation = create_dim_backshift_permutation(dim_[0], dim_[1], self.dim());
   auto permutation_reverse = create_reverse_permutation(permutation);
 
   Tensor p = self.permute(permutation);
   at::sum_out(result, std::get<1>(at::svd(p, /*some=*/true, /*compute_uv=*/false)), -1, keepdim);
   if (keepdim) {
     result.unsqueeze_(-1);
-    result = result.permute(permutation_reverse);
+    Tensor result_ = result.permute(permutation_reverse);
+    result.resize_as_(result_);
```

Comment:
Nit: instead of resizing directly this should call

https://github.com/pytorch/pytorch/blob/dae94ed02299cd1e9840ce6e49ea142b4d118b2d/aten/src/ATen/native/Resize.cpp#L9